### PR TITLE
Update rubygem-foreman_resource_quota to 0.3.1

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec [hunk 1]
```diff
@@ -18,13 +18,14 @@ BuildRequires: foreman-plugin >= %{foreman_min_version}
...
+BuildRequires: (rubygem(foreman-tasks) >= 10.0 with rubygem(foreman-tasks) < 11)
 BuildArch: noarch
 Provides: foreman-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
 
 # start package.json devDependencies BuildRequires
-BuildRequires: (npm(@babel/core) >= 7.7.0 with npm(@babel/core) < 8.0.0)
-BuildRequires: npm(@theforeman/builder) >= 12.0.1
+BuildRequires: (npm(@babel/core) >= 7.26.0 with npm(@babel/core) < 8.0.0)
+BuildRequires: npm(@theforeman/builder) >=12.0.1
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec:28]:
```suggestion
BuildRequires: npm(@theforeman/builder) >= 12.0.1
```
RPM is picky :/

**Review by evgeni — CHANGES_REQUESTED** [review]

File: packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec (hunk 1 above)
Comment (nadjaheitmann) [reply packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec:28]: Yeah, thanks, I have already opened a PR in the plugin itself. I guess, we need to fix it there. I don't feel like manually adjusting it with every packaging PR :)

Comment (nadjaheitmann) [reply packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec:28]: https://github.com/ATIX-AG/foreman_resource_quota/pull/82

Comment (evgeni) [reply packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec:28]: Thanks, but we can fix it here and don't need a new release just for that :)

Comment (nadjaheitmann) [reply packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec:28]: OK, then let's fix it here and have it fixed for the new release

**Comment by nadjaheitmann** [comment]
There seems to be more amiss than just spec file linting :roll_eyes:

**Comment by evgeni** [comment]
```
 Problem: nothing provides requested (npm(@babel/core) >= 7.26.0 with npm(@babel/core) < 8.0.0)
```

:thinking: 

We only have 7.23.9 packaged right now :cry:

**Comment by nadjaheitmann** [comment]
> ```
>  Problem: nothing provides requested (npm(@babel/core) >= 7.26.0 with npm(@babel/core) < 8.0.0)
> ```
> 
> 🤔
> 
> We only have 7.23.9 packaged right now 😢
> 
> And #10519 has been failing forever :(

@bastian-src I think we should stop using dependabot :upside_down_face:

**Review by evgeni — APPROVED** [review]

**Pull request merged**